src/main/java/core/stategies/ServerStrategyContext.java

```diff
@@ -11,8 +14,8 @@ public void setStrategy(ServerStrategy strategy) {
         this.strategy = strategy;
     }
 
-    public void execute(SocketPayload socketPayload, List<Client> clientList) {
-        Client client = this.strategy.processAuthor(socketPayload, clientList);
+    public void execute(SocketPayload socketPayload, List<ClientHandler> clientList) {
```

Comment:
function below should not return boolean because it's unused value, xsan fucked up this ( ͡° ͜ʖ ͡°)